src/components/JsLogo.vue

```diff
@@ -28,6 +39,9 @@ export default {
       } else {
         return '../static/logos/' + this.currentLogo.name.toLowerCase() + '.png'
       }
+    },
+    altContext: function () {
+      return this.gameFinished ? 'Learn about this logo library' : 'Guess this logo'
```

Comment:
Can you change `Learn about this logo library` to `Learn about this JavaScript library`